playbooks/ansible-cloud/k8s/run.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: all
+  vars:
+    ansible_zuul_src_dir: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/ansible/ansible-zuul-jobs'].src_dir }}"
+  tasks:
```

Comment:
Can we use tox here? This was we would be able to delegate the venv creation and the molecule execution to tox and keep the configuration (tox.ini and test-requirements.txt) within the repositor(y|ies) that we test.